Sources/Property.swift

```diff
@@ -371,6 +371,25 @@ extension PropertyProtocol {
 	}
 }
 
+extension PropertyProtocol where Value == Bool {
+	/// Returns a property with the oposite value
+	public var not: Property<Value> {
```

Comment:
`negated` perhaps. It is not a boolean but a container of booleans, and properties should be named as nouns whenever possible.